easybuild/easyconfigs/s/starcode/starcode-1.4-GCC-11.2.0.eb

```
@@ -0,0 +1,43 @@
+# easybuild easyconfig
+#
+# John Dey <jfdey@fredhutch.org>
+#
+# Fred Hutchinson Cancer Center - Seattle Washington US
+#
+easyblock = 'MakeCp'
+
+name = 'starcode'
+version = '1.4'
+
+github_account = 'gui11aume'
+homepage = 'https://github.com/%(github_account)s/starcode'
+description = """Starcode is a DNA sequence clustering software. Starcode clustering is based on all pairs
+search within a specified Levenshtein distance (allowing insertions and deletions), followed by a clustering
+algorithm: Message Passing, Spheres or Connected Components."""
+
+toolchain = {'name': 'GCC', 'version': '11.2.0'}
+
+dependencies = [
+]
+
+source_urls = ['https://github.com/%(github_account)s/%(name)s/archive/refs/tags']
+sources = ['%(version)s.tar.gz']
+patches = ['%(name)s_extern.patch']
+checksums = [
+    'b4f0eae9498f2dcf9c8a7fa2fa88b141c6d5abcf6da44b03d5d85e5f1a8fd5b1',  # 1.4.tar.gz
+    '09334952cca76593d864cacd397d1daba19ba6ab2861a054e164b931ce100c87',  # starcode_extern.patch
```

I assume you you added the description to the patch after injecting the checkum into the easyconfig:
```suggestion
    '1d62d75e26f4c136d85819810c4f02c45de5e0f26d3ac3be7de1c8a0185c1875',  # starcode_extern.patch
```